src/client/pages/OfferNew/utils.ts

```diff
@@ -243,6 +244,27 @@ export const apartmentTypeTextKeys: Record<ApartmentType, string> = {
   [ApartmentType.StudentBrf]: 'CHECKOUT_INSURANCE_APARTMENT_TYPE_BRF',
 }
 
+export const getInsuranceTitle = (
+  offerData: OfferData,
+  textKeys: TextKeyMap,
+) => {
+  if (!isBundle(offerData)) {
+    return textKeys[insuranceTypeTextKeys[offerData.quotes[0].contractType]]()
+  }
+
+  if (isNorwegian(offerData)) {
+    return textKeys.SIDEBAR_INSURANCE_TYPE_NO_BUNDLE()
+  }
+
+  if (isDanish(offerData)) {
+    return offerData.quotes.length === 2
+      ? textKeys.SIDEBAR_INSURANCE_TYPE_DK_CONTENTS_ACCIDENT()
+      : textKeys.SIDEBAR_INSURANCE_TYPE_DK_CONTENTS_ACCIDENT_TRAVEL()
+  }
+
+  return ''
```

Comment:
And what about:
```suggestion
  if (isBundle(offerData) && isNorwegian(offerData)) {
    return textKeys.SIDEBAR_INSURANCE_TYPE_NO_BUNDLE()
  }

  if (isBundle(offerData) && isDanish(offerData)) {
    return offerData.quotes.length === 2
      ? textKeys.SIDEBAR_INSURANCE_TYPE_DK_CONTENTS_ACCIDENT()
      : textKeys.SIDEBAR_INSURANCE_TYPE_DK_CONTENTS_ACCIDENT_TRAVEL()
  }

  return textKeys[insuranceTypeTextKeys[offerData.quotes[0].contractType]]()
```

Reply:
Not super important, just feels more... logical(?) to me to have the text key for a single quote as default return, instead of an empty string 🙊

Reply:
that works as well! 👌 I think I switched 3 or 4 different solutions before commiting 😄 I was thinking if we should add `isNorwegianBundle` and `isDanishBundle` to utils but maybe that feels redundant or WDYT? @styrelius

Reply:
Haha I also started to write something about having `isNorwegianBundle` and `isDanishBundle` util functions first but then also felt that it's probably overkill 😄 Or will we use this in many places?

Reply:
Haha, yeah let's keep it simple then and add it later if we feel like it 🙌